packages/gatsby-theme-learning/src/hooks/use-course-status.ts

```diff
@@ -0,0 +1,59 @@
+import useSWR from 'swr';
```

Comment:
Note about SWR: by default `revalidateIfStale = true` meaning that even if swr returns the stale data straight away, it will perform a fetch in the background to refresh the state, in case it changed. So, don't get fooled by the network activity (like me), stale data is actually being returned

Reply:
Yes, you just "see" it as the status not disappearing but staying the old status while the new state is fetched.